### ISSUE TITLE
[BUG] Regression in 3.21

### ISSUE THREAD

**Issue by yubiuser** [mborsetti/webchanges#76]
```
2024-04-17 15:16:54,066 handler[140439589538616] INFO: Job 18: https://hub.docker.com/v2/repositories/library/docker/tags/?page_size=100&page=1&name&ordering ended processing
Traceback (most recent call last):
  File "webchanges/handler.py", line 165, in process
  File "webchanges/jobs.py", line 1006, in retrieve
  File "webchanges/jobs.py", line 765, in _retrieve_httpx
webchanges.jobs.NotModifiedError: 304

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "webchanges/handler.py", line 180, in process
2024-04-17 15:16:54,074 filters[140439585246008] INFO: Job 11: Applying filter element-by-id, subfilter(s) {'id': 'news'}
  File "webchanges/jobs.py", line 1043, in format_error
AttributeError: 'str' object has no attribute 'exceptions'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "webchanges.py", line 4, in <module>
  File "webchanges/cli.py", line 392, in main
  File "webchanges/command.py", line 1099, in run
```

```
name: Docker cli alpine images
url: https://hub.docker.com/v2/repositories/library/docker/tags/?page_size=100&page=1&name&ordering
filter:
     - jq: '.results | sort_by(.name) | .[] |  select(.name | contains("alpine")) | .name'
```

**Comment by mborsetti** [comment]
Sorry, I somehow missed this one.

Thanks for the report, and apologies; will fix in the next few hours.

**Comment by mborsetti** [comment]
Fixed in 3.22, just released.
